quickstart/docker/image/run-controller.sh

```diff
@@ -7,8 +7,11 @@ export ZITI_EDGE_CONTROLLER_RAWNAME=ziti-edge-controller
 
 . "${ZITI_SCRIPTS}/ziti-cli-functions.sh"
 
-generateEnvFile
-. ${ZITI_HOME}/ziti.env
+# access-control.init is created by ziti-controller-init-container after initial controller startup
+if [ ! -f "${ZITI_HOME}/access-control.init" ]; then
```

Comment:
does this work with "just docker"? aka not compose too?

Reply:
It does, assuming they follow the directions. If they don't run the init-container (step 2 in "With Docker" setup) then it would not.

I toyed with the idea of seeing if the .env file was empty or not to determine the password generation. Would you prefer that method? Or, we can have a file generated by the controller, just like `access-control.init` and just keep the same style as above with the controller-based file instead.